pkg/simple/client/auditing/elasticsearch/elasticsearch.go

```diff
@@ -360,6 +360,36 @@ func parseToQueryPart(f *auditing.Filter) interface{} {
 		return &bi
 	}
 
+	if len(f.ObjectRefNamespaces) > 0 {
+		if bi := shouldBoolbody("match_phrase_prefix", "ObjectRef.Namespace.keyword",
```

Comment:
match_phrase_prefix should be change to match_phrase to be more accurate for both namespace and workspace query